common/djangoapps/student/management/tests/test_bulk_user_activate.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for the bulk_user_activate command."""
+
+import ddt
```

Comment:
ddt isn't really needed since we just need to test the functionality of the command with a domain once